tests/test_handlers.py

```diff
@@ -1131,7 +1131,7 @@ async def test_cache_flush_bit():
     for record in new_records:
         assert zc.cache.async_get_unique(record) is not None
 
-    original_a_record.created = current_time_millis() - 1001
```

Comment:
This one wasn't fast enough and sometimes we would outrun the clock in the test and it would fail so 1.5s to split the difference instead